pytorch/pytorch-native/build.gradle

```diff
@@ -76,22 +77,25 @@ task releaseJNI {
     doFirst {
         // You have to use an environment with CUDA persets for Linux and Windows
         if (System.properties['os.name'].toLowerCase(Locale.ROOT).contains("windows")) {
-            downloadBuild("win", "cpu")
-            cleanBuild()
-            downloadBuild("win", "cu101")
-            cleanBuild()
-            downloadBuild("win", "cu102")
-            cleanBuild()
-            downloadBuild("win", "cu110")
-            cleanBuild()
-        } else if (System.properties['os.name'].toLowerCase(Locale.ROOT).contains("mac")) {
+            if (!isGPU) {
```

Comment:
This is just optimization, right?

Reply:
releaseJNI used to compile cpu and gpu. If we split upload jobs, there might be a chance where both task uploads to the same file to s3